sdk/core/Azure.Core.Experimental/src/JsonData.cs

```diff
@@ -674,62 +508,62 @@ public override DynamicMetaObject BindSetIndex(SetIndexBinder binder, DynamicMet
             }
         }
 
-        internal class JsonDataDebuggerProxy
-        {
-            [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-            private readonly JsonData _jsonData;
-
-            public JsonDataDebuggerProxy(JsonData jsonData)
-            {
-                _jsonData = jsonData;
-            }
-
-            [DebuggerDisplay("{Value.DebuggerDisplay,nq}", Name = "{Name,nq}")]
-            internal class PropertyMember
-            {
-                [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-                public string? Name { get; set; }
-                [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
-                public JsonData? Value { get; set; }
-            }
-
-            [DebuggerDisplay("{Value,nq}")]
-            internal class SingleMember
-            {
-                public object? Value { get; set; }
-            }
-
-            [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
-            public object Members
-            {
-                get
-                {
-                    if (_jsonData.Kind != JsonValueKind.Array &&
-                        _jsonData.Kind != JsonValueKind.Object)
-                        return new SingleMember() { Value = _jsonData.ToJsonString() };
-
-                    return BuildMembers().ToArray();
-                }
-            }
-
-            private IEnumerable<object> BuildMembers()
-            {
-                if (_jsonData.Kind == JsonValueKind.Object)
-                {
-                    foreach (var property in _jsonData.Properties)
-                    {
-                        yield return new PropertyMember() { Name = property, Value = _jsonData.Get(property) };
-                    }
-                }
-                else if (_jsonData.Kind == JsonValueKind.Array)
-                {
-                    foreach (var property in _jsonData.Items)
-                    {
-                        yield return property;
-                    }
-                }
-            }
-        }
+        //internal class JsonDataDebuggerProxy
```

Comment:
These will be added back last.  The methods that only debugger view calls into don't need to be as performant as those used by applications at runtime.

Reply:
Tracking with https://github.com/Azure/azure-sdk-for-net/issues/33856